webpack.config.js

```diff
@@ -13,13 +13,19 @@ var path = require('path');
                  test: /\.js$/,
                  loader: 'babel-loader',
                  query: {
-                     presets: ['es2015', 'react']
+                     presets: ['es2015','stage-0', 'react']
```

Comment:
I noticed the presets exist both here and in the `.babelrc` file, do they serve different purposes?